src/chrome/content/rules/WellsFargo.xml

```diff
@@ -1,11 +1,15 @@
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://wellsfargo.com/ => https://www.wellsfargo.com/: (28, 'Operation timed out after 0 milliseconds with 0 out of 0 bytes received')
--->
-<ruleset name="Wells Fargo" default_off='failed ruleset test'>
-  <target host="wellsfargo.com" />
-  <target host="*.wellsfargo.com" />
+<ruleset name="Wells Fargo">
+	<target host="wellsfargo.com" />
+	<target host="*.wellsfargo.com" />
 
-  <rule from="^http://wellsfargo\.com/" to="https://www.wellsfargo.com/" />
-  <rule from="^http://([^/:@\.]+)\.wellsfargo\.com/" to="https://$1.wellsfargo.com/" />
+	<securecookie host="(^|\.)wellsfargo\.com$"
+			name=".+" />
+
+	<test url="http://wellsfargo.com/" />
+	<test url="http://www.wellsfargo.com/" />
+
+	<rule from="^http://wellsfargo\.com/"
+		to="https://www.wellsfargo.com/" />
+	<rule from="^http://([^/:@\.]{1,})\.wellsfargo\.com/"
```

Comment:
The new preferred style for this is `([\w-]+)`. Any reason for this particular style?